mama/c_cpp/src/c/subscription.c

```diff
@@ -452,6 +452,7 @@ mamaSubscription_setupBasic (
     }
 
 #ifdef WITH_ENTITLEMENTS
+	{
```

Comment:
Really wish these were submitted before the publisher events stuff was started - these ones in particular will make merging the entitlements changes a nightmare. You sure there are no command line options you can use to disable the mixed declaration enforcement in the interim? This has all been compiled on windows before.